content/post/blue-boot2-root-writeup.md

```diff
@@ -0,0 +1,699 @@
+[*] Nmap: |   OS: Windows 7 Professional 7601 Service Pack 1 (Windows 7 Professional 6.1)
+[*] Nmap: |   OS CPE: cpe:/o:microsoft:windows_7::sp1:professional
+[*] Nmap: |   Computer name: Jon-PC
+[*] Nmap: |   NetBIOS computer name: JON-PC\x00
+[*] Nmap: |   Workgroup: WORKGROUP\x00
+[*] Nmap: |_  System time: 2020-11-10T19:54:50-06:00
+[*] Nmap: | smb-security-mode:
+[*] Nmap: |   account_used: guest
+[*] Nmap: |   authentication_level: user
+[*] Nmap: |   challenge_response: supported
+[*] Nmap: |_  message_signing: disabled (dangerous, but default)
+[*] Nmap: | smb2-security-mode:
+[*] Nmap: |   2.02:
+[*] Nmap: |_    Message signing enabled but not required
+[*] Nmap: | smb2-time:
+[*] Nmap: |   date: 2020-11-11T01:54:50
+[*] Nmap: |_  start_date: 2020-11-11T01:51:00
+[*] Nmap: Service detection performed. Please report any incorrect results at https://nmap.org/submit/ .Nmap done: 1 IP address (1 host up) scanned in 140.63 seconds
+```
+
+This initial scan tended to provide further evidence toward my beliefs that this was an eternalblue attack as it looked like the host was a Windows 7 Service Pack 1 machine with SMB available. In addition, this ended up leaking a potential username as the computer's name, which was worth keeping in mind as a potential administrative user going forward. Additionally this pointed out relaxed security setting in the smb service's configuration.
+
+Give what I knew so far I decided to probe a little further into SMB with an enumaration using nmap's `smb-enum-shares` scripts.
+
+```
+msf5 > db_nmap --script smb-enum-shares  -p 445 10.10.201.83
+[*] Nmap: Starting Nmap 7.80 ( https://nmap.org ) at 2020-11-11 01:58 UTC
+[*] Nmap: Nmap scan report for ip-10-10-201-83.eu-west-1.compute.internal (10.10.201.83)
+[*] Nmap: Host is up (0.00020s latency).
+[*] Nmap: PORT    STATE SERVICE
+[*] Nmap: 445/tcp open  microsoft-ds
+[*] Nmap: MAC Address: 02:B5:23:84:9D:19 (Unknown)
+[*] Nmap: Host script results:
+[*] Nmap: | smb-enum-shares:
+[*] Nmap: |   note: ERROR: Enumerating shares failed, guessing at common ones (NT_STATUS_ACCESS_DENIED) |   account_used: <blank>
+[*] Nmap: |   \\10.10.201.83\ADMIN$:
+[*] Nmap: |     warning: Couldn't get details for share: NT_STATUS_ACCESS_DENIED
+[*] Nmap: |     Anonymous access: <none>
+[*] Nmap: |   \\10.10.201.83\C$:
+[*] Nmap: |     warning: Couldn't get details for share: NT_STATUS_ACCESS_DENIED
+[*] Nmap: |     Anonymous access: <none>
+[*] Nmap: |   \\10.10.201.83\IPC$:
+[*] Nmap: |     warning: Couldn't get details for share: NT_STATUS_ACCESS_DENIED
+[*] Nmap: |_    Anonymous access: READ
+[*] Nmap: Nmap done: 1 IP address (1 host up) scanned in 0.85 seconds
+```
+
+Again this pointed to anonymous read access on the IPC share, another pointer towards eternalblue being the vector. With this information, I decided I should kick off a quick vuln scan with `nmap` to see if it would infact identify eternalblue. Which it quickly did, as `ms17-010`.
```

Comment:
> Again this pointed to anonymous read access on the IPC share, another pointer towards eternalblue being the vector.

This indicated anonymous read access on the IPC share, another piece of evidence pointing to eternalblue as being the attack vector